pkg/framework/aries/framework_test.go

```diff
@@ -175,6 +179,74 @@ func TestFramework(t *testing.T) {
 		require.NoError(t, err)
 	})
 
+	t.Run("test protocol svc - with default protocol", func(t *testing.T) {
+		aries, err := New()
+		require.NoError(t, err)
+		require.NotEmpty(t, aries)
+
+		ctx, err := aries.Context()
+		require.NoError(t, err)
+		exist := false
+		for _, v := range ctx.RESTHandlers() {
+			if v.Path() == "/create-invitation" {
+				exist = true
+				break
+			}
+		}
+		require.True(t, exist)
+
+		err = aries.Close()
+		require.NoError(t, err)
+	})
+
+	t.Run("test protocol svc - with user provided protocol", func(t *testing.T) {
+		newMockSvc := func(prv api.Provider) (api.ProtocolSvc, error) {
+			return mockProtocolSvc{getAPIHandlersValue: []api.Handler{mockHandler{path: "testPath"}}}, nil
```

Comment:
No need of MockHandler,
You can reuse : https://github.com/hyperledger/aries-framework-go/blob/master/pkg/internal/common/support/httphandler.go#L15

Reply:
done